scripts/gis-material-update/validate-deploy/validate_deploy_data.py

```diff
@@ -0,0 +1,73 @@
+"""Main entrypoint script for material processing.
+"""
+import os
+import sys
+
+from modules.config import Config
+from modules.gis_validate_deploy import GisProcessor
+
+from modules.autoliikennemaarat import MakaAutoliikennemaarat
+from modules.hsl import HslBuses
+from modules.ylre_katualueet import YlreKatualueet
+from modules.ylre_katuosat import YlreKatuosat
+from modules.tram_infra import TramInfra
+from modules.tram_lines import TramLines
+from modules.cycling_infra import CycleInfra
+from modules.liikennevaylat import Liikennevaylat
+from modules.central_business_area import CentralBusinessAreas
+
+DEFAULT_DEPLOYMENT_PROFILE = "local_development"
+
+def validate_deploy_item(item: str, cfg: Config):
+    print(f"Validating deployment item: {item}")
+    gis_processor = instantiate_processor(item, cfg)
+    gis_processor.validate_deploy()
+    pass
+
+def instantiate_processor(item: str, cfg: Config) -> GisProcessor:
+    """Instantiate correct class for processing data."""
+    if item == "hsl":
+        return HslBuses(cfg)
+    elif item == "maka_autoliikennemaarat":
+        return MakaAutoliikennemaarat(cfg)
+    elif item == "ylre_katuosat":
+        return YlreKatuosat(cfg)
+    elif item == "ylre_katualueet":
+        return YlreKatualueet(cfg)
+    elif item == "tram_infra":
+        return TramInfra(cfg)
+    elif item == "tram_lines":
+        return TramLines(cfg)
+    elif item == "cycle_infra":
+        return CycleInfra(cfg)
+    elif item == "liikennevaylat":
+        return Liikennevaylat(cfg)
+    elif item == "central_business_area":
+        return CentralBusinessAreas(cfg)
+    else:
+        try:
+            raise RuntimeError("Configuration not recognized: {}".format(item))
+        except Exception as e:
+            print("{}".format(e))
```

Comment:
Why is there a try that always throws an exception?

Reply:
Is source given as parameter for gis-validate-deploy? And is the source one of supported sources?

Reply:
Is there a difference between raising an exception in a `try` and printing it in an except as opposed to just printing it directly? I don't know of any, but I'm not very familiar with Python.

```suggestion
        print("{}".format(RuntimeError("Configuration not recognized: {}".format(item))))
```

If there's no difference, we could skip the exception altogether and just print the error directly.
```python
        print("Configuration not recognized: {}".format(item))
```

Reply:
Changed